flink-dstl/flink-dstl-dfs/src/main/java/org/apache/flink/changelog/fs/FsStateChangelogWriter.java

```diff
@@ -199,15 +200,21 @@ public SequenceNumber nextSequenceNumber() {
         return activeSequenceNumber;
     }
 
+    @VisibleForTesting
+    CompletableFuture<SnapshotResult<ChangelogStateHandleStreamImpl>> persist(SequenceNumber from)
```

Comment:
I think it is better to let the test code adapt to the interface change.

Reply:
The tests related to the Local file have been modified accordingly, for other tests I left them as they are.

Reply:
I think all of that should be modified. Since that all test code can easily adapt to the new interface, I don't see the need to keep this method for test code alone. In my opinion this breaks the legibility of the code.